docs/docs/contributing/intellij-idea.md

```diff
@@ -20,40 +20,51 @@ its installed on your local machine). Otherwise, specify it by pressing *New*.
 
 ![](../../images/idea/idea-sdk.png "Select the JDK")
 
-On the final window we must select which modules we can import. Here, we are presented with the full list of SBT projects
+In some versions of IDEA, we must select which modules we can import. Here, we are presented with the full list of SBT projects
```

Comment:
If you have that handy can you give a "v > XX.XX" format example of which versions?

Reply:
Good question, but is there any reason why we should currently support multiple IDEA versions? Supporting only one (in the docs) seems a reasonable tradeoff.

Maybe people who don't want to upgrade can install a new version next to the other.

Gotta setup IDEA (already run into bad bugs from ScalaIDE), so I can take a look at this PR.

Reply:
After some testing, I found that the change happened in version 2017.2 released in November 2017.

@Blaisorblade You have a good point in general, but I can think of two reasons to support multiple versions. First, the change was made a few months ago, so many people will need the first directions. Second, this page is unique in that its main purpose is to make on-boarding as frictionless as possible. That seems to call for setting aside the general principle of supporting only the newest version that applies elsewhere.